test/e2e/docker/Dockerfile

```diff
@@ -1,10 +1,18 @@
 # We need to build in a Linux environment to support C libraries, e.g. RocksDB.
 # We use Debian instead of Alpine, so that we can use binary database packages
-# instead of spending time compiling them.
+# instead of spending time compiling them. 
+# -> There is currently no librocksdb-dev v6.17.x that is necessary by line/gorocksdb.
```

Comment:
`v6.17.x and more` instead of `v6.17.x` since we use `rocksdb-v6.20.3`?

Reply:
I don't understand what you mean.

Reply:
I understood that `line/gorocksdb` need `librocksdb-dev v6.17.x`, but docker task is defined:
```
`RUN wget -O rocksdb-v6.20.3.tar.gz`
```

I've just confused this sentence.
```
There is currently no librocksdb-dev v6.17.x that is necessary by line/gorocksdb.
```